src/AWS.Deploy.Recipes/RecipeDefinitions/BlazorWasm.recipe

```diff
@@ -9,6 +9,14 @@
     "CdkProjectTemplateId": "netdeploy.BlazorWasm",
     "Description": "Deploy a Blazor WebAssembly application to an Amazon S3 bucket. The Amazon S3 bucket created during deployment will be configured for web hosting and its contents will be open to the public with read access.",
     "TargetService": "Amazon S3",
+
+    "DisplayedResources": [
+        {
+            "LogicalId": "BlazorHostC7106839",
```

Comment:
Is this logical id generated?

Reply:
Yes it is generated. https://quip-amazon.com/JwTMAqlP7vE4/Deploy-Tool-Output#cMA9CAvyoVu